### PR TITLE
LPD-29610 Users cannot go back to the root object when inside a second-level child

### PR DESCRIPTION
**Pull request by pedro-oliveira446**
[LPD-29610](https://liferay.atlassian.net/browse/LPD-29610)

[LPD-29610]: https://liferay.atlassian.net/browse/LPD-29610?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by pedro-oliveira446** [comment]
ci:test:sf

**Comment by pedro-oliveira446** [comment]
ci:test:sf

**Review by MarinhoFeliphe — CHANGES_REQUESTED** [review]
Good job @pedro-oliveira446!!

see my comments

**Comment by pedro-oliveira446** [comment]
ci:test:sf

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

**Comment by pedro-oliveira446** [comment]
ci:test:object

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/object/entries/display/context/test/ObjectEntryDisplayContextTest.java [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+			"1", _objectEntryLocalService, _objectFieldLocalService,
+			nodeA.getPrimaryKey(), _objectRelationshipLocalService,
+			_treeFactory);
+
+		ObjectDefinition objectDefinitionAA =
+			_objectDefinitionLocalService.getObjectDefinition(
+				TestPropsValues.getCompanyId(), "C_AA");
+
+		ObjectEntry objectEntryAA = _objectEntryLocalService.getObjectEntry(
+			"AA1", objectDefinitionAA.getObjectDefinitionId());
```
Comment (MarinhoFeliphe) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/object/entries/display/context/test/ObjectEntryDisplayContextTest.java:82]: Hey @pedro-oliveira446, just rename all objectEntries of this hierarchy to `objectEntryAA1`, `objectEntryA1`, ..

thanks!

**Review by MarinhoFeliphe — APPROVED** [review]
LGTM!! :tada:

Well done, @pedro-oliveira446 just apply my suggestion and forward it.

**Comment by pedro-oliveira446** [comment]
ci:test:sf

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

**Comment by pedro-oliveira446** [comment]
ci:test:object

**Comment by pedro-oliveira446** [comment]
ci:test:sf

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

**Comment by pedro-oliveira446** [comment]
ci:test:object

**Comment by pedro-oliveira446** [comment]
ci:forward

**Pull request closed without merging**